NDB.Covid19/NDB.Covid19.Droid/Resources/layout/layout_with_launcher_button_ag_api.xml

```diff
@@ -1,90 +1,93 @@
-<LinearLayout
-	xmlns:android="http://schemas.android.com/apk/res/android"
-	android:orientation="vertical"
-	android:layout_width="match_parent"
-	android:layout_height="match_parent"
-	android:gravity="center_horizontal"
-	>
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Please add scroll view as now this may not fit the smaller screens

Reply:
It has been tested to work on an emulator as small as 4'' how small devices are we supporting?

Reply:
but have you tested also with accessibility test size as big as possible?

Reply:
On this device it looks like this with largest text size: ![Screenshot_1606745505](https://user-images.githubusercontent.com/11331332/100620166-631fdc80-331e-11eb-87b9-8a52a6f58355.png)

Reply:
Display size is on max on this device

Reply:
ok, so i guess it is ok then